src/app/create-account/create-account.component.html

```diff
@@ -178,14 +176,59 @@ <h1> Register </h1>
                 <input style="height:30px;width:425px;" formControlName="confirmPassword" id="confirmPassword" type="password" placeholder="Confirm password">
               </div>
             </div>
-          </form>
           <div class="button">
             <button class="create-account-button" type="button" (click)="createAccount()">
               Create Account
             </button>
           </div>
         </div>
-      </div>
-    </div>
+    </form>
   </div>
 </div>
+-->
+<!--Attempt 3 (I promise it's the final!)-->

```

Comment:
Attempt 3 looks good, so you can delete the first 2 attempts

Reply:
I will!